deploy.sh

```diff
@@ -152,8 +152,12 @@ svn update
 
 svn status
 
-echo "➤ Committing files..."
-svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+if [[ "$DEBUG" != false ]]; then
```

Comment:
This if looks backwards – currently it commits files when `DEBUG` is truthy/not-false which is the opposite of what I think we want 😄 

I think the general logic is what we want though `if debug` do dry run stuff, otherwise do live run – it's just the bodies of the conditions that needs to change.

Also, maybe a non-zero string condition would be more appropriate? `if [[ -n "$DRY_RUN" ]]` I'm not sure how bash evaluates a string to `!= false` – does that treat `1` as true and `0` as false?

Reply:
Thanks @aaemnnosttv for feedback.

I have updated the conditions in https://github.com/10up/action-wordpress-plugin-deploy/pull/122/commits/f484e0e8503d198527df7b4c4b614ffc122285e8 and tested the workflow in the folk repository. Here is the result.

- If `DRY_RUN` not set: [Live mode](https://github.com/mukeshpanchal27/performance/actions/runs/4685562391/jobs/8302778766)
- If `DRY_RUN` set to `false`: [Live mode](https://github.com/mukeshpanchal27/performance/actions/runs/4685558010/jobs/8302769563)
- If `DRY_RUN` set to anything else ex. `testing`: [Live mode](https://github.com/mukeshpanchal27/performance/actions/runs/4685579676/jobs/8302812758)
- If `DRY_RUN` set to `true`: [Debug mode](https://github.com/mukeshpanchal27/performance/actions/runs/4685646100/jobs/8302947105)

Reply:
In https://github.com/10up/action-wordpress-plugin-deploy/pull/122/commits/2311fb74e077648c58a2b58da5d06703a02af91d i have updated conditions as per the feedback.